libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -434,6 +434,10 @@ void AP_CRSF_Telem::process_packet(uint8_t idx)
             calc_flight_mode();
             break;
         case PASSTHROUGH:
+            //first push lua data (if available), then other telem data
```

Comment:
#if AP_SCRIPTING_ENABLED
need a custom frame id
there needs to be a way of switching this off. Please add an option bit to RC_OPTIONS, default off

Reply:
I don't think an additional RC_OPTIONS bit is required. The bidirectional telemetry option is switched off by default, only after the first Lua call to crsfPop/Push the pop/push buffers are allocated and the bidirectional telemetry becomes active.

Reply:
Hi, I like the general idea, very nice!
but I'm afraid you won't get much bandwidth the way you designed it, you'll get a chance of sending a packet FC->TX only at "passthrough scheduler rate" which is quite low, around 10Hz, and not any faster (the scheduler has a rate limit) so your actual througput will be lower than expected, on ELRS we can probably go faster but the scheduler rates are designed to be compatible with both ELRS and CRSF.
Also we do not shut standard CRSF telemetry off so quite some bandwidth is used for legacy CRSF telemetry frames as well. 
Your code will starve passthrough packets and be delayed by other CRSF streams, a better approach IMO would be to declare an additional scheduler stream with a potentially much higher priority, this way you would get a bigger chance of sending packets without actually starving any stream since the packet scheduler is "fair" by design

Reply:
Hi yaapu, thanks! It was your excellent telemetry script that inspired me to start this PR, in order to eventually also get your ground station script to work with elrs.

I'm aware of the scheduler issues, but did not feel comfortable at this point to mess around with it. The code works as is, but there is definitely room for improvement!

Reply:
Wouldn't it be a clean solution to add Lua functions crsfSetFreq(Hz) and crsfStandardTele(on|off) to control this?